Xcode/xcconfigs/dylib.xcconfig

```diff
@@ -16,3 +16,6 @@ DYLIB_COMPATIBILITY_VERSION = 1
 DYLIB_CURRENT_VERSION       = 1
 DYLIB_INSTALL_NAME_BASE     = @rpath
 EXECUTABLE_PREFIX           = lib
+
+CODE_SIGN_IDENTITY = Apple Development
+DEVELOPMENT_TEAM = N2Q372V7W2
```

Comment:
Might be good to align the = indentation with the others

Reply:
As you suggested, I actually turned off code signing.